build/git.go

```diff
@@ -48,9 +48,14 @@ func getGitAttributes(ctx context.Context, contextPath string, dockerfilePath st
 		wd, _ = filepath.Abs(filepath.Join(cwd, contextPath))
 	}
 
-	gitc := gitutil.New(gitutil.WithContext(ctx), gitutil.WithWorkingDir(wd))
+	gitc, err := gitutil.New(gitutil.WithContext(ctx), gitutil.WithWorkingDir(wd))
+	if err != nil {
+		logrus.Warnf("Failed to initialize git: %v", err)
+		return
+	}
+
 	if !gitc.IsInsideWorkTree() {
```

Comment:
False in here should not be a warning. If we get error from `git` then we should show it. Or say that `--debug` is needed to see the error.